### PR TITLE
Fix dead link about Cyrus, mention NeoMutt fork with WIP JMAP implementation

### PR DESCRIPTION
**Pull request by timofonic**
Hello.

The changes are simple. I hope they are okay.

Kind regards.

### CODE REVIEW

File: software/software.mdown [hunk 1]
```diff
@@ -25,13 +26,3 @@ JMAP is still young but several implementations already exist, and more are on t
...
-
-## Requested
-
-* [RainLoop Webmail](https://github.com/RainLoop/rainloop-webmail/issues/1378)
-* [Cypht](https://github.com/jasonmunro/cypht/issues/180)
-* [Horde IMP Webmail](https://bugs.horde.org/ticket/14683)
-
-
-
-
```
Comment (neilj) [inline software/software.mdown]: Did you mean to remove the "Requested" section?

File: software/software.mdown [hunk 2]
```diff
@@ -5,12 +5,13 @@ JMAP is still young but several implementations already exist, and more are on t
 ## Clients
 
 * **[JMAP Demo Webmail](https://github.com/jmapio/jmap-demo-webmail)** (JavaScript, MIT): a sophisticated demo webmail client to be used as a base for new projects. Default client for the [proxy](https://proxy.jmap.io).
-* **[JMAP::Tester](https://metacpan.org/pod/JMAP::Tester)** (Perl, Perl5): a JMAP client made for testing JMAP servers
+* **[JMAP::Tester](https://metacpan.org/pod/JMAP::Tester)** (Perl, Perl5): a JMAP client made for testing JMAP servers.
+* **[NeoMutt](https://github.com/robn/neomutt/tree/jmap)** (C): A command-line mail reader. It's a version of Mutt with added features. An experimental fork with WIP JMAP support is available.
```
Comment (robn) [inline software/software.mdown]: I'd prefer not to mention this fork. It was an experiment that I am unlikely to continue with (at least not before a proper JMAP library for C exists).

Comment (timofonic) [reply software/software.mdown]: @robn Okay. I'm sorry. You're right. But the Cyrus link didn't work and I updated it. Sorry if I broke it. You can update the Cyrus Mail link and close this :)

File: software/software.mdown (hunk 1 above)
Comment (timofonic) [reply software/software.mdown]: I broke the file, sorry.

**Comment by neilj** [comment]
OK, I've merged the Cyrus change.

**Pull request closed without merging**